doc/classes/TextureLayered.xml

```diff
@@ -1,10 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TextureLayered" inherits="Texture" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Base class for 3D texture types.
+		Base class for texture types which contain the data of multiple [Image]s. Each image is of the same size and format.
 	</brief_description>
 	<description>
-		Base class for [Texture2DArray], [Cubemap] and [CubemapArray]. Cannot be used directly, but contains all the functions necessary for accessing the derived resource types. Data is set on a per-layer basis. For [Texture2DArray]s, the layer specifies the array layer.
+		Base class for [Texture2DArray], [Cubemap] and [CubemapArray]. Cannot be used directly, but contains all the functions necessary for accessing the derived resource types.
+		Data is set on a per-layer basis. For [Texture2DArray]s, the layer specifies the array layer.
+		All images need to have the same width, height and number of mipmap levels.
+		A [TextureLayered] can be loaded with [code]method ResourceFormatLoader.load[/code].
+		To create such a texture file yourself, re-import your image files using the Godot Editor import presets.
+		Alternatively, see the instructions of the thirdparty library Godot uses: https://github.com/BinomialLLC/basis_universal.
```

Comment:
This can be deleted. User's won't be using basis universal directly

Reply:
removed